Formula/eigenpy.rb

```diff
@@ -0,0 +1,48 @@
+class Eigenpy < Formula
+  desc "Python bindings of Eigen library with Numpy support"
+  homepage "https://github.com/stack-of-tasks/eigenpy"
+  url "https://github.com/stack-of-tasks/eigenpy/releases/download/v2.6.6/eigenpy-2.6.6.tar.gz"
+  sha256 "d5ece10d1c255de746352b5d225b7216d258fb3f0df0d0fb39e0c86d8b6a0dfa"
+  license "BSD-2-Clause"
+  head "https://github.com/stack-of-tasks/eigenpy.git", branch: "master"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "boost-python3"
+  depends_on "eigen"
+  depends_on "numpy"
+  depends_on "python@3.9"
+
+  def install
+    pyver = Language::Python.major_minor_version "python3"
+    py_prefix = Formula["python@3.9"].opt_bin
+    ENV.prepend_path "PYTHONPATH", Formula["numpy"].opt_prefix/Language::Python.site_packages("python3")
+    ENV.prepend_path "Eigen3_DIR", Formula["eigen"].opt_share/"eigen3/cmake"
+
+    mkdir "build" do
+      args = *std_cmake_args
+      args << "-DPYTHON_EXECUTABLE=#{py_prefix}/bin/python#{pyver}"
```

Comment:
```suggestion
    python = Formula["python@#{pyver}"].opt_bin/"python#{pyver}"
    ENV.prepend_path "PYTHONPATH", Formula["numpy"].opt_prefix/Language::Python.site_packages("python3")
    ENV.prepend_path "Eigen3_DIR", Formula["eigen"].opt_share/"eigen3/cmake"

    mkdir "build" do
      args = *std_cmake_args
      args << "-DPYTHON_EXECUTABLE=#{python}"
```
You've duplicated the `bin` directory here. Let's try to make this simpler.